<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlastHitFiltersDialog</class>
 <widget class="QDialog" name="BlastHitFiltersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BLAST hit filters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="InfoTextWidget" name="eValueInfoText" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="InfoTextWidget" name="alignmentLengthInfoText" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="InfoTextWidget" name="queryCoverageInfoText" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDoubleSpinBox" name="eValueCoefficientSpinBox">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="maximum">
         <double>9.990000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="eValueExponentSpinBox">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="prefix">
         <string>e</string>
        </property>
        <property name="minimum">
         <number>-999</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="InfoTextWidget" name="identityInfoText" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="InfoTextWidget" name="bitScoreInfoText" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="identitySpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="bitScoreCheckBox">
     <property name="text">
      <string>Bit score:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="alignmentLengthSpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="queryCoverageSpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="queryCoverageCheckBox">
     <property name="text">
      <string>Query coverage:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="eValueCheckBox">
     <property name="text">
      <string>E-value:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="alignmentLengthCheckBox">
     <property name="text">
      <string>Alignment length:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="identityCheckBox">
     <property name="text">
      <string>Identity:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="filterDescriptionLabel">
     <property name="text">
      <string>Turn on one or more of these filters to exclude BLAST hits which fail to meet the defined threshold. To see the effects of changing these filters, it is necessary to rerun the BLAST search.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QDoubleSpinBox" name="bitScoreSpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InfoTextWidget</class>
   <extends>QWidget</extends>
   <header>infotextwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>alignmentLengthCheckBox</tabstop>
  <tabstop>alignmentLengthSpinBox</tabstop>
  <tabstop>queryCoverageCheckBox</tabstop>
  <tabstop>queryCoverageSpinBox</tabstop>
  <tabstop>identityCheckBox</tabstop>
  <tabstop>identitySpinBox</tabstop>
  <tabstop>eValueCheckBox</tabstop>
  <tabstop>eValueCoefficientSpinBox</tabstop>
  <tabstop>eValueExponentSpinBox</tabstop>
  <tabstop>bitScoreCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BlastHitFiltersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BlastHitFiltersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
